copter/source/docs/systemid-mode-ctrl-optimization.rst

```diff
@@ -0,0 +1,257 @@
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Bandwidth (Hz)                         | 0.73      | **1.84**  | 1.58     | Larger is better  |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Gain Margin (dB) at f_pc               | **25.92** | 11.44     | 2.35     | Larger is better  |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Phase Margin (deg) at f_gc             | 13.27     | **60.63** | 7.97     | Larger is better  |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Phase crossover frequency f_pc (Hz)    | 3.01      |    5.9    | 3.01     | neutral           |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Gain crossover frequency f_gc (Hz)     | 0.55      |  **1.84** | 1.76     | Larger is better  |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Tracking Rise Time (s)                 | 0.47      | 0.214     | **0.21** | Smaller is better |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Tracking Overshoot (%)                 | 17.16     | **0.00**  | 3.96     | Smaller is better |
++----------------------------------------+-----------+-----------+----------+-------------------+
+| Tracking Settling Time (s)             | 3.25      | **1.42**  | 1.65     | Smaller is better |
++----------------------------------------+-----------+-----------+----------+-------------------+
+
+The following two tables show the optimized flight controller parameters for the roll and yaw axis compared to the default parameters of ArduCopter.
+The defaults are also used as initial parameters for the optimization.
+As a reference, parameters obtained from an autotune flight with :ref:`AUTOTUNE_AGGR<AUTOTUNE_AGGR>` = 0.1 are also shown.
+It shows that the optimized parameters differ greatly from the default and autotune parameters.
+Again remember that the presented results are **only valid for the examined quadcopter** with a weight of 1.5 kg, an arm length of 22 cm and 9 inch propellers.
+The resulting parameters are specifically applicable for this copter configuration and **must not** be used for any other copter.
+
+Roll:
+
++-------------------------------------------+-----------------------+----------------------+------------------------+
+| Parameter                                 | Default               | Optimized            | Autotune               |
++-------------------------------------------+-----------------------+----------------------+------------------------+
+| :ref:`ATC_RAT_RLL_P<ATC_RAT_RLL_P>`       | 0.1350                | 0.123                | 0.240025               |
```

Comment:
can I see a log of it flying with I = 8x P ?

Reply:
I believe this is caused by the high disturbance rejection criteria.   I would think this could have some implications with integrator management during sloped takeoffs and landings.